apps/docs/pages/guides/getting-started/local-development.mdx

```diff
@@ -418,9 +418,10 @@ To use Auth locally, update your project's `supabase/config.toml` file that gets
 ```bash config.toml
 [auth.external.github]
 enabled = true
-client_id = ""
-secret = ""
+client_id = "env(GITHUB_CLIENT_ID)"
+secret = "env(GITHUB_SECRET)"
```

Comment:
We should probably guide users with a way to set and load these (ie. `.env` file). Thoughts on best way to show that?

Reply:
Yes I agree. I feel the documentation I'm linking to is already pretty decent. However, I noticed that the when deploying the db changes and project via GH actions the env files from the config are required as well.  (we should add that to the docs as well)

Since I'm still a new user of Supabase, I'm actually not sure what the best pattern is. Should users ideally use the local development environment as the full representation of the prod/hosted project, or should they only use it for prototyping and having a throw away copy of the prod db? In the latter case we'd probably want every dev to have their own custom config.toml and not commit them to git, right? 

Otherwise, I guess it would make sense for us to expand on the idea of configuring and using the config.toml as the full config you'd otherwise manually do in the admin interface.

Reply:
`.env` files aren't required when using GH actions as you set the environment variables inside of the GitHub UI itself. `.env` files should only ever be used for local development and not CI/CD (GitHub actions in this scenario) or production. Your `config.toml` can be comitted to git as long as you are using the `env()` function in them, the `.env` file is whhat you wouldn't commit.

Reply:
Yes exactly. You should just know that you'll have to both set up the environment variables in CI/CD as well as locally through `.env`.

In case the config is committed to git, does the `supabase link` or `supabase db push` command apply any changes to the `config.toml` file as well?

Reply:
Okay, I just checked the CLI code and it seems like the CLI doesn't apply changes from the config to a supabase-hosted project. I created an [Issue in the CLI repo](https://github.com/supabase/cli/issues/1202) to suggest that the config parser should ignore unset environment variables for these commands. 

@gregnr @silentworks what would you guys recommend as for the recommended usage to document? Should we simply dissuade users from committing their config.toml to git - thus avoiding any issues with accidentally disclosing the auth secrets? Or should we recommend the full implementation with .env files and GItHub secrets (even if the GH secrets setup is actually not needed)